spec/Gist_Request_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative '../Gist_Request.rb'
+require 'spec_helper'
+
+RSpec.describe GistRequest do
+  describe "#post" do
+    it "campos no válidos" do
```

Comment:
Here:
```
it "should return 422 if fields are invalid" do
    ...
end
```

Important: if your testing the method `post` (which you are, according to your `describe`), you should call that method within the test.